runtime/binding-tls/src/test/java/io/aklivity/zilla/runtime/binding/tls/internal/config/TlsConditionConfigAdapterTest.java

```diff
@@ -72,4 +72,41 @@ public void shouldWriteCondition()
         assertThat(text, not(nullValue()));
         assertThat(text, equalTo("{\"authority\":\"example.net\",\"alpn\":\"echo\"}"));
     }
+
+    @Test
+    public void shouldReadConditionWithPortRange()
+    {
+        String text =
+                "{" +
+                    "\"authority\": \"example.net\"," +
+                    "\"alpn\": \"echo\"," +
+                    "\"port\": 8080-8081" +
+                "}";
+
+        TlsConditionConfig condition = jsonb.fromJson(text, TlsConditionConfig.class);
+
+        assertThat(condition, not(nullValue()));
+        assertThat(condition.ports, not(nullValue()));
+        assertThat(condition.ports.length, equalTo(2));
+        assertThat(condition.ports[0], equalTo(8080));
+        assertThat(condition.ports[1], equalTo(8081));
+    }
+
+    @Test
+    public void shouldReadConditionWithPortRangeSingleton()
+    {
+        String text =
+                "{" +
+                    "\"authority\": \"example.net\"," +
+                    "\"alpn\": \"echo\"," +
+                    "\"port\": \"8080\"" +
+                "}";
+
+        TlsConditionConfig condition = jsonb.fromJson(text, TlsConditionConfig.class);
+
+        assertThat(condition, not(nullValue()));
+        assertThat(condition.ports, not(nullValue()));
+        assertThat(condition.ports.length, equalTo(1));
+        assertThat(condition.ports[0], equalTo(8080));
+    }
```

Comment:
Please add `shouldWriteConditionWithPortRangeSingleton()` test method, see `shouldWriteConfition()` for an example.